umh.docs.umh.app/content/en/docs/production-guide/security/firewall-rules.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
There was some additional behavior, that we need to document and could result in problems:

By default for the docker registries docker, quay and ghcr it first tries to fetch it from management.umh.app, and then from the original registry. All other registries are disabled by default and can only be fetched from management.umh.app.

This means using any other docker registry in custom applications that the user might deploy, would fail. To mitigate this, the user would need to adjust the file `/var/lib/rancher/k3s/agent/etc/containerd/config.toml.tmpl` # See https://github.com/k3s-io/k3s/issues/2088 for more context


          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
          endpoint = ["https://management.umh.app/oci", "https://registry-1.docker.io"]
        
          # Mirror configuration for GitHub Container Registry with fallback
          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."ghcr.io"]
          endpoint = ["https://management.umh.app/oci", "https://ghcr.io"]
        
          # Mirror configuration for Quay with fallback
          [plugins."io.containerd.grpc.v1.cri".registry.mirrors."quay.io"]
          endpoint = ["https://management.umh.app/oci", "https://quay.io"]